<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSet</class>
 <widget class="QWidget" name="TimeSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>УКМ Установка времени (revision 003)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="allBox">
     <property name="title">
      <string>Установка времени</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QDateEdit" name="otherHostDate"/>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="otherHostTime"/>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="otherHostRadio">
        <property name="text">
         <string>Время вручную</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDateEdit" name="curHostDate">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="curHostTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="curHostRadio">
        <property name="text">
         <string>Время данного компьютера</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="setButton">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
